parser-typechecker/src/Unison/Server/CodebaseServer.hs

```diff
@@ -113,6 +115,24 @@ type AuthedServerAPI = ("static" :> Raw) :<|> (Capture "token" Text :> ServerAPI
 instance ToSample Char where
   toSamples _ = singleSample 'x'
 
+-- BaseUrl and helpers
+
+data BaseUrl = BaseUrl { urlHost :: String
+                       , urlToken :: Strict.ByteString
+                       , urlPort :: Port
+                       }
```

Comment:
We want to start shifting to using `ormolu` for formatting, specifically for new or changed code.  Can you set it up in your editor and "format region" here, and possibly other modified sections.  I wouldn't hold up the PR for it, but we do want to start switching and gotta start somewhere.  OTOH, I also wouldn't reformat all the files on a feature PR because it will make it hard to review the diff.

Reply:
Reformatted with Ormolu.

Reply:
It looks like we missed the `ui` `InputPattern` but we'll get it eventually.

Reply:
@aryairani whats the impact of missing that?

Reply:
@hojberg I meant it was just another region of code to auto-format, the impact is low.